docs/onboarding/rpc/wallet.md

```diff
@@ -114,9 +119,11 @@ Creates and returns a new transaction with the given parameters, but doesn't pos
 }
 `} />
 
-## wallet/exportAccount
+## <GithubCodeLink link="wallet/exportAccount"/> wallet/exportAccount
+
+Exports the keys to the default account, or the named account if specified. 
 
-Exports the keys to the default account, or the named account if specified. If `viewOnly` is true, the spending key will be null, but the spending key may also be null if exporting a view-only account.
+If `viewOnly = true`, the returned spending key will be null, but the spending key may also be null if exporting a view-only account.
```

Comment:
The wording was already like this, but I think it might be better as:

```suggestion
`spendingKey` will be null in the response if `viewOnly = true` or if exporting a view-only account.
```